packages/block-editor/src/store/test/actions.js

```diff
@@ -770,7 +770,21 @@ describe( 'actions', () => {
 	describe( 'toggleBlockMode', () => {
 		it( 'should return TOGGLE_BLOCK_MODE action', () => {
```

Comment:
Should we add tests for the cases where either `codeEditingEnabled` or `hasBlockSupport( ... )` is falsey?